pyspark/bigdl/nn/keras/layer.py

```diff
@@ -64,7 +66,36 @@ def jvm_class_constructor(self):
         return name
 
 
-class Sequential(TSequential, InferShape):
+class Training(JavaValue):
+    def __init__(self, bigdl_type="float"):
+        self.bigdl_type = bigdl_type
+
+    def compile(self, optimizer, loss, metrics=None):
+        init_engine()
```

Comment:
Wondering where shall we put `init_engine`, `redirect_spark_logs` and `show_bigdl_info_logs` as in the new API these should be transparent from the user.